tabs/src/components/code/component/text/Toolbar.jsx

```diff
@@ -0,0 +1,151 @@
+import React from "react";
+import { Toolbar } from '@fluentui/react-northstar'
+import { BoldIcon, ItalicIcon, UnderlineIcon, FontSizeIcon, RemoveFormatIcon } from '@fluentui/react-northstar'
+import { OutdentIcon, IndentIcon, MoreIcon, LinkIcon, CodeSnippetIcon, QuoteIcon } from '@fluentui/react-northstar'
```

Comment:
They are all from @fluentui/react-northstar, so you can make them in one import sentence

Reply:
ok, I will add them in one import sentence and split them to lines